.github/workflows/perf-testing-setup.yml

```diff
@@ -2,12 +2,11 @@ name: Throughput testing workflow
 
 on:
   workflow_dispatch:
-  pull_request:
```

Comment:
Can we also run this pipeline on a comment?
Something like :
on:
  issue_comment:
  types:
    - created

Reply:
updated to run on comment contains "/profile"